README.md

```diff
@@ -17,3 +20,7 @@ The following packet is broadcast over RS232 at a rate of 115200, 8 data bits, n
 The checksum is the sum of bytes 0-25.
 
 The byte ordering is in big endian.
+
+## CAN Simulator (SimCan)
+
+Currently supports sending out 2 messages on fixed interval. 
```

Comment:
It might be useful to include some wiring instructions here for people working on this in the future. This part of the telemetry stack is pretty encapsulated from the rest so it'd make sense to document it here IMO.

Up to you as to whether you want to include it in this PR though.

Reply:
What do you mean by wiring instructions? If you mean a schematic, I'm actually working on that and I do plan to include it in this repo

Reply:
Yup, schematics or even a textual description would be great.

There used to be a telemetry project before I joined the team but it didn't go anywhere, and all of that knowledge was lost to the wind since the information was scattered across random Google Drive files. One of the things I was hoping to consolidate was documentation so that it's easier for new members to pick up old projects.